frontends/election-manager/src/components/election_manager_tally_report.tsx

```diff
@@ -1,298 +1,74 @@
-    }
-    if (batchId) {
-      return `${statusPrefix} Batch Tally Report for ${batchLabel}`;
-    }
-    if (precinctIdFromProps === 'all') {
-      return `${statusPrefix} ${election.title} Tally Reports for All Precincts`;
-    }
-    if (partyIdFromProps) {
-      const party = find(election.parties, (p) => p.id === partyIdFromProps);
-      return `${statusPrefix} Tally Report for ${party.fullName}`;
-    }
-    if (votingMethod) {
-      const label = getLabelForVotingMethod(votingMethod);
-      return `${statusPrefix} ${label} Ballot Tally Report`;
-    }
-    return `${statusPrefix} ${election.title} Tally Report`;
-  }, [
-    batchId,
-    scannerId,
-    precinctIdFromProps,
-    votingMethod,
-    partyIdFromProps,
-    batchLabel,
-    statusPrefix,
-    election,
-    precinctName,
-  ]);
+export interface Props {
+  batchId?: string;
+  batchLabel?: string;
+  election: Election;
+  fullElectionExternalTallies: FullElectionExternalTally[];
+  fullElectionTally: FullElectionTally;
+  generatedAtTime?: Date;
+  isOfficialResults: boolean;
+  partyId?: string;
+  precinctId?: string;
+  scannerId?: string;
+  votingMethod?: VotingMethod;
+}
 
-  useEffect(() => {
-    if (showPreview) {
-      void logger.log(LogEventId.TallyReportPreviewed, currentUserType, {
-        message: `User previewed a ${reportDisplayTitle} to view.`,
-        disposition: 'success',
-        tallyReportTitle: reportDisplayTitle,
-      });
-    }
-  }, [showPreview, logger, reportDisplayTitle, currentUserType]);
+export const ElectionManagerTallyReport = forwardRef<HTMLDivElement, Props>(
+  (
+    {
+      batchId,
+      batchLabel,
+      election,
+      fullElectionExternalTallies,
+      fullElectionTally,
+      generatedAtTime = new Date(),
+      isOfficialResults,
+      partyId: partyIdFromProps,
+      precinctId: precinctIdFromProps,
+      scannerId,
+      votingMethod,
+    },
+    ref
+  ) => {
+    const statusPrefix = isOfficialResults ? 'Official' : 'Unofficial';
 
-  function afterPrint() {
-    void logger.log(LogEventId.TallyReportPrinted, currentUserType, {
-      message: `User printed ${reportDisplayTitle}`,
-      disposition: 'success',
-      tallyReportTitle: reportDisplayTitle,
-    });
-  }
+    const ballotStylePartyIds =
+      partyIdFromProps !== undefined
+        ? [unsafeParse(PartyIdSchema, partyIdFromProps)]
+        : Array.from(new Set(election.ballotStyles.map((bs) => bs.partyId)));
 
-  function afterPrintError(errorMessage: string) {
-    void logger.log(LogEventId.TallyReportPrinted, currentUserType, {
-      message: `Error in attempting to print ${reportDisplayTitle}: ${errorMessage}`,
-      disposition: 'failure',
-      tallyReportTitle: reportDisplayTitle,
-      errorMessage,
-      result: 'User shown error.',
-    });
-  }
+    const precinctIds =
+      precinctIdFromProps === 'all'
+        ? election.precincts.map((p) => p.id)
+        : [precinctIdFromProps];
```

Comment:
The logic to generate `statusPrefix`, `ballotStylePartyIds`, and `precinctIds` isn't new. Looking at the diff without whitespace changes ([using `?w=1`](https://github.com/votingworks/vxsuite/pull/1701/commits/47f67e85eed54117e5d47162873a9859c167395b?w=1)) makes this clear